prereise/gather/demanddata/transportation_electrification/smart_charging.py

```diff
@@ -137,7 +103,15 @@ def get_constraints(constraints_df, kwhmi, power, trip_strategy, location_strate
 
 
 def calculate_optimization(
-    charging_consumption, rates, elimit, seg, segsum, segcum, total_trips, kwh
+    charging_consumption,
+    rates,
+    elimit,
+    seg,
+    segsum,
+    segcum,
+    total_trips,
+    kwh,
+    charging_efficiency,
 ):
```

Comment:
why do we need to pass `segsum` and `cumsum`? These could easily be calculated in the function:
```
segsum = np.sum(seg)
segcum = np.cumsum(seg)
```